R/tests/testthat/helper-sqlPackage.R

```diff
@@ -108,22 +108,22 @@ helper_CreateExternalLibrary <- function(connectionString, packageName, authoriz
 
     if (substr(content, 0, 2) == "0x")
     {
-        createExtLibDDLString = paste0(createExtLibDDLString, " FROM (content = ", content, ") WITH (LANGUAGE = 'R')")
+        createExtLibDDLString = paste0(createExtLibDDLString, " FROM (content = ", content, ") WITH (LANGUAGE = '", languageName,"')")
     }
     else
     {
-        createExtLibDDLString = paste0(createExtLibDDLString, " FROM (content = '", content, "') WITH (LANGUAGE = 'R')")
+        createExtLibDDLString = paste0(createExtLibDDLString, " FROM (content = '", content, "') WITH (LANGUAGE = '", languageName,"')")
     }
 
     helper_ExecuteSQLDDL(connectionString = connectionString, sqlDDL = createExtLibDDLString)
 }
 
-helper_callDummySPEES <- function(connectionString)
+helper_callDummySPEES <- function(connectionString, languageName="R")
 {
     cat(sprintf("\nINFO: call dummy sp_execute_external_library to trigger install.\r\n"))
-    speesStr = "EXECUTE sp_execute_external_script
-    @LANGUAGE = N'R',
-    @SCRIPT = N'invisible(NULL)'"
+    speesStr = paste0("EXECUTE sp_execute_external_script

```

Comment:
Against which environment do the new language extension tests run ? 
The reason i ask is, since Azure SQL MI doesn't support language extensions, we would need to test out against box instance for language extensions ?

Reply:
there are no tests for the extensions right now. We will have to reevaluate the tests later.

Honestly, E2E testing in Github is difficult and I'm not sure I actually want to continue doing that here. We may want to make some unit tests that just compare the created scripts vs expected scripts, but for now i am not changing it.

Reply:
oh I see. Then you are manual testing these changes ?


---
In reply to: [502705244](https://github.com/microsoft/sqlmlutils/pull/82#discussion_r502705244) [](ancestors = 502705244)

Reply:
yes

---
In reply to: [502719207](https://github.com/microsoft/sqlmlutils/pull/82#discussion_r502719207) [](ancestors = 502719207,502705244)